test/integration/bigip_device_auth_radius.yaml

```diff
@@ -0,0 +1,52 @@
+---
+
+# Test the bigip_device_auth_radius module
+#
+# Running this playbook assumes that you have a BIG-IP installation at the
+# ready to receive the commands issued in this Playbook.
+#
+# This module will run tests against a BIG-IP host to verify that the
+# bigip_device_auth_radius module behaves as expected.
+#
+# Usage:
+#
+#    ansible-playbook -i notahost, test/integration/bigip_device_auth_radius.yaml
+#
+# Examples:
+#
+#    Run all tests on the bigip_device_auth_radius module
+#
+#    ansible-playbook -i notahost, test/integration/bigip_device_auth_radius.yaml
+#
+
+- name: Metadata of bigip_device_auth_radius
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    __metadata__:
+      upstream: no
```

Comment:
this needs to be set to yes, otherwise the module will not be in collection

Reply:
This comment is addressed.